### ISSUE TITLE
Delay JavaScript Execution compatibility with WP iCal Availability

### ISSUE THREAD

**Issue by DahmaniAdame** [wp-media/wp-rocket#4696]
**Before submitting an issue please check that you’ve completed the following steps:**
- [x] Made sure you’re on the latest version `3.10.7`
- [x] Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
When DJE is enabled and `jquery` is excluded, WP iCal Availability will return the follow errors: 

```
jquery.min.js?ver=3.6.0:2 jQuery.Deferred exception: wpia(...).customSelect is not a function TypeError: wpia(...).customSelect is not a function
    at HTMLDocument.<anonymous> (/wp-content/plugins/wp-ical-availability/js/wpia.js?ver=5.9:4:56)
    at e (/wp-includes/js/jquery/jquery.min.js?ver=3.6.0:2:30038)
    at t (/wp-includes/js/jquery/jquery.min.js?ver=3.6.0:2:30340) undefined
S.Deferred.exceptionHook @ jquery.min.js?ver=3.6.0:2
t @ jquery.min.js?ver=3.6.0:2
setTimeout (async)
(anonymous) @ jquery.min.js?ver=3.6.0:2
c @ jquery.min.js?ver=3.6.0:2
add @ jquery.min.js?ver=3.6.0:2
(anonymous) @ jquery.min.js?ver=3.6.0:2
s.Deferred.s.Deferred @ jquery-migrate.min.js?ver=3.3.2:2
then @ jquery.min.js?ver=3.6.0:2
S.fn.ready @ jquery.min.js?ver=3.6.0:2
(anonymous) @ wpia.js?ver=5.9:4
jquery.min.js?ver=3.6.0:2 Uncaught TypeError: wpia(...).customSelect is not a function
    at HTMLDocument.<anonymous> (wpia.js?ver=5.9:4:56)
    at e (jquery.min.js?ver=3.6.0:2:30038)
    at t (jquery.min.js?ver=3.6.0:2:30340)
```

Excluding the following from DJE fixes the issue: 

```
/wp-ical-availability/js/custom-select.js
```

**To Reproduce**
1. Install [WP iCal Availability](https://wordpress.org/plugins/wp-ical-availability/), link a calendar to it, and add its shortcode to a page
2. Enable DJE and exclude `jquery`
3. See error

**Expected behavior**
There shouldn't be any errors when using the WP iCal Availability, even after excluding `jquery`. 

**Screenshots**
N/A

**Additional context**
Ticket - https://secure.helpscout.net/conversation/1761857621/320269/

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

**Comment by piotrbak** [comment]
Will be fixed with the 3.13 backend

**Comment by piotrbak** [comment]
Handled in 3.13 using dynamic lists
